src/test/fuzz.zig

```diff
@@ -0,0 +1,47 @@
+//! Utils functions for writing fuzzers.
+
+const std = @import("std");
+const assert = std.debug.assert;
+
+/// Returns an integer of type `T` with an exponential distribution of rate `avg`.
+/// Note: If you specify a very high rate then `std.math.maxInt(T)` may be over-represented.
+pub fn random_int_exp(random: std.rand.Random, comptime T: type, avg: T) T {
+    comptime {
+        const info = @typeInfo(T);
+        assert(info == .Int);
+        assert(info.Int.signedness == .unsigned);
+    }
+    const exp = random.floatExp(f64) * @intToFloat(f64, avg);
+    return std.math.lossyCast(T, exp);
+}
+
+/// Return a list of probabilities for use with `random_enum_biased`.
+pub fn random_biases(
```

Comment:
Would `random_biased_distribution` be an accurate name? To group with `random_biased_enum` below.

Reply:
How about `random_enum_distribution` and `random_enum`?

Reply:
Perfect, even better.